lab_1_classify_by_unigrams/main.py

```diff
@@ -11,14 +11,36 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
 
+    list_of_tokens = ""
+    for token in text:
```

Comment:
try to simplify the function

Reply:
fixed

Reply:
You can make list without intermediate variables. Use list comprehension

Reply:
got it now, thank you! fixed